datafusion/physical-plan/src/topk/mod.rs

```diff
@@ -212,15 +212,56 @@ impl TopK {
             })
             .collect::<Result<Vec<_>>>()?;
 
+        let mut selected_rows = None;
```

Comment:
Does the fact that we have removed this code mean we don't have `TopK` pushdown filtering happening on main anymore? 

- I am trying to understand performance results I am seeing in https://github.com/apache/datafusion/pull/16562

Reply:
I believe we still have some effect: the dynamic filter is pushed down into the scan. What we are now missing is essentially https://github.com/apache/datafusion/pull/15697